### PR TITLE
Use Manage Nav

### PR DESCRIPTION
**Pull request by ColinMcNeil**
# Release Notes

Added UCONN CSS to navbars, as well as sidenav. For best results enable use/manage mode.

# Screenshot
## Use Mode
screen shot 2018-11-09 at 9 34 09 am

## Manage Mode

screen shot 2018-11-09 at 9 34 21 am

## Sidenav 
screen shot 2018-11-09 at 9 34 30 am

### CODE REVIEW

File: app/assets/stylesheets/nucore.scss [hunk 1]
```diff
@@ -408,3 +438,191 @@ textarea.wide {
...
+  color: $useColorDark;
+  transition: ease 0.3s all;
+}
+
+.manage-mode .container > .nav > li > a , .manage-mode .nav.nav-tabs > li > a{
+  color: $manageColorDark;
+}
+
+.container > .nav > li:hover > a {
+  text-shadow: 0 0 0.5px $useColorDark;
```
Comment (jhanggi) [inline app/assets/stylesheets/nucore.scss]: Is it intentional to use the orange as the text-shadow color, even in manage mode?

screen shot 2018-11-13 at 4 59 05 pm

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -4,13 +4,16 @@ $baseFontSize: 12px;
...
 @import "bootstrap-responsive";
+
 @import "font-awesome";
 @import "chosen";
 
 @import "nucore-variables";
 @import "nucore-overrides";
 @import "nucore";
 
+@import "responsive";
```
[the remark this answers is not in the document]
Comment (ColinMcNeil) [reply app/assets/stylesheets/application.scss]: This one is stumping me. I didn't change any behavior with the nav responsiveness.  It should collapse to the small menu just like before. I also can't figure out how on earth `responsive.scss` is getting imported in the original branch.

[the remark this answers is not in the document]
Comment (jhanggi) [reply app/assets/stylesheets/application.scss]: It's brought in as a separate `<script>` tag in `views/shared/_head.html.haml` based on `responsive?`, which is in `ApplicationHelper`.

I don't think we ever did the responsive/hamburger menu for the admin side.

[the remark this answers is not in the document]
Comment (ColinMcNeil) [reply app/assets/stylesheets/application.scss]: Oh, I see that now. Looks like in manage mode there is no primary nav.

**Comment by jhanggi** [comment]
Looks like there's still some funkiness with the nav at small widths. The use/manage buttons aren't looking right within the hamburger menu, and things are overlapping on the manage views in the nav.

I'm ok if we merge this as is and then fix it in a separate PR. Whatever works best for you.

screen shot 2018-11-20 at 11 32 14 am
screen shot 2018-11-20 at 11 31 47 am

**Comment by ColinMcNeil** [comment]
I was able to make it work well using some media queries. There might be a better way but it would involve more time/energy. Should be ready to merge.

**Comment by jhanggi** [comment]
Menu looks better in manage mode. The use/manage buttons in the hamburger menu dropdown are still a little funky, but I'll let you take care of that later.

**Pull request merged**